classes/author.rb

```diff
@@ -0,0 +1,14 @@
+class Author
+  attr_reader :id, :first_name, :last_name, :items
+
+  def initialize(id, first_name, last_name, items = [])
+    @id = id
+    @first_name = first_name
+    @last_name = last_name
+    @items = items
+  end
+
+  def add_item(item)
+    @items << item
```

Comment:
Could you please add this line after line 12:
`item.add_author(self)`

This is needed because when you add the item to your Author you also need to assign an author to the item.
I know that item.rb doesn't have `add_author` method yet, this is what I'm going to ask Essohanam to change today.